Normalize style in makeprop.pl's output to aid replacing it

weinig opened the pull request:
#### 4718faa509f327097a30847ee21ede844aff6670
<pre>
Normalize style in makeprop.pl&apos;s output to aid replacing it
<a href="https://bugs.webkit.org/show_bug.cgi?id=247550">https://bugs.webkit.org/show_bug.cgi?id=247550</a>
rdar://102015817

Reviewed by Darin Adler.

To make replacing makeprop.pl easier in <a href="https://bugs.webkit.org/show_bug.cgi?id=247470">https://bugs.webkit.org/show_bug.cgi?id=247470</a>
this updates the generated output to consistently use the following stylist choices:

- Always use fully scoped name for CSSPropertyIDs (this will also make it a bit
easier to move to a scoped enum at some point).
- Use the parameter name &quot;id&quot; for CSSPropertyID parameters in generated functions.
- Move all the fullly static function bodies generated for CSSPropertyNames.gperf to
be consecutive.
- Add a missing space in the static part of isInheritedPropertyTable so that all
the true&apos;s look the same.

* Source/WebCore/css/makeprop.pl:
(generateFillLayerPropertyValueSetter):
(generateValueSetter):

Canonical link: <a href="https://commits.webkit.org/256379@main">https://commits.webkit.org/256379@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/6e921760048836090aec989fb2f87defd96a30c4

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/95617 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/4880 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/28665 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/105196 "Built successfully") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/165489 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/99603 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/4935 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/33631 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/87989 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/101046 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/101279 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/3612 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/82233 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/30676 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/85485 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/87395 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/73512 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/39368 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/18946 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/37068 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/20252 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/4408 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/41064 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/42900 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/43052 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/39502 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

darinadler commented on Source/WebCore/css/makeprop.pl:
```diff
@@ -518,7 +518,54 @@ sub sortWithPrefixedPropertiesLast
+    return value ? static_cast<CSSPropertyID>(value->id) : CSSPropertyID::CSSPropertyInvalid;
+}
+
+ASCIILiteral nameLiteral(CSSPropertyID id)
```
My least favorite part of this is calling everything an "id". I wish the type name and the argument names didn’t use that terminology.

Commit:
Normalize style in makeprop.pl's output to aid replacing it
https://bugs.webkit.org/show_bug.cgi?id=247550
rdar://102015817

Reviewed by Darin Adler.

To make replacing makeprop.pl easier in https://bugs.webkit.org/show_bug.cgi?id=247470
this updates the generated output to consistently use the following stylist choices:

- Always use fully scoped name for CSSPropertyIDs (this will also make it a bit
easier to move to a scoped enum at some point).
- Use the parameter name "id" for CSSPropertyID parameters in generated functions.
- Move all the fullly static function bodies generated for CSSPropertyNames.gperf to
be consecutive.
- Add a missing space in the static part of isInheritedPropertyTable so that all
the true's look the same.

* Source/WebCore/css/makeprop.pl:
(generateFillLayerPropertyValueSetter):
(generateValueSetter):

Canonical link: https://commits.webkit.org/256379@main
